### PR TITLE
DX-98; allow disabling prettyPrint using env var

### PR DESCRIPTION
**Pull request by schehata**

### CODE REVIEW

File: src/shared.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ export const proxyPath = '/_axiom';
 
 export const isBrowser = typeof window !== 'undefined';
 export const isEnvVarsSet = process.env.AXIOM_INGEST_ENDPOINT || process.env.NEXT_PUBLIC_AXIOM_INGEST_ENDPOINT;
+export const isPrettyPrintEnabled = process.env.AXIOM_PRETTY_PRINT_ENABLED == 'false' ? false : true;
```
Comment (bahlo) [inline src/shared.ts:5]: Do you think using `NO_COLOR` instead would make more sense? https://no-color.org/

Comment (bahlo) [inline src/shared.ts:5]: I wonder if we should instead use `AXIOM_NO_PRETTY_PRINT` and check for a truthy value, wdyt?

**Review by bahlo — APPROVED** [review]
LGTM otherwise

File: src/shared.ts (hunk 1 above)
Comment (schehata) [reply src/shared.ts]: done 👍

**Pull request merged**